compile_assets/scss/admonition.scss

```
@@ -186,6 +188,30 @@ a.admonition-anchor-link {
   }
 }
 
+summary {
```

We should be specific we're only styling _our_ summaries here, there could be others on the page in user content.

For example:

```suggestion
summary.admonition-title {
```